### PR TITLE
Only flag :1 or :3 slices

### LINKED ISSUE #3: False positive with version slice
the version slice should probably only check for `:1` and `:3` as there's a few "legitimate" usages of slicing `sys.version` I've seen in the wild, in particular here's one from cpython that seems to be well defined:

https://github.com/python/cpython/blob/da27d9b9dc44913ffee8f28d9638985eaaa03755/Lib/distutils/msvc9compiler.py#L174-L179

### PR DESCRIPTION
**Pull request by mxr**
Resolves #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ you might also find an early build of [python3.10] useful
 
 | Code   | Description                                            |
 |--------|--------------------------------------------------------|
-| YTT101 | `sys.version[:...]` referenced (python3.10)            |
+| YTT101 | `sys.version[:1]` referenced (python3.10)              |
```
Comment (hugovk) [inline README.md]: Is this a `python10` issue?

Comment (mxr) [reply README.md]: Good call. I can reword `YTT101` to only reference `sys.version[:3]` and introduce `YTT303` for `:1`?

Comment (asottile) [reply README.md]: Yeah let's do that

Comment (mxr) [reply README.md]: OK updated

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ you might also find an early build of [python3.10] useful
 | YTT204 | `sys.version_info.minor` compared to integer (python4) |
 | YTT301 | `sys.version[0]` referenced (python10)                 |
 | YTT302 | `sys.version` compared to string (python10)            |
+| YTT303 | `sys.version[:1]` referenced (python310)               |
```
Comment (asottile) [inline README.md]: python10

Comment (mxr) [reply README.md]: I can write a code generator for this / copy one of the ones from pygrep-hooks or another repo

Comment (asottile) [reply README.md]: ooh yeah that would be nice -- the one in `pygrep-hooks` is a pretty decent example

Comment (mxr) [reply README.md]: Fixed manually for now

**Pull request merged**